benches/all.rs

```diff
@@ -36,6 +36,43 @@ macro_rules! bench_chars {
     };
 }
 
+fn std_from_byte_idx(text: &str, mut byte_idx: usize) -> usize {
```

Comment:
I think we can inline this in the test code, by just using a `{ }` code block inside the parenthesis of the `black_box` call.